x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/ilm/ExecuteStepsUpdateTaskTests.java

```diff
@@ -140,17 +140,17 @@ private IndexMetadata setupIndexPolicy(String policyName) {
         lifecycleState.setPhase("new");
         lifecycleState.setAction("init");
         lifecycleState.setStep("init");
-        IndexMetadata indexMetadata = IndexMetadata.builder(indexName)
+        IndexMetadata _indexMetadata = IndexMetadata.builder(indexName)
```

Comment:
We don't typically use underscore prefixed variable names, do we?

Reply:
We don't, no. I think I just ran out of inspiration here, but I went back and changed it 👍